modules/tinymce/src/core/main/ts/api/dom/DOMUtils.ts

```diff
@@ -1804,15 +1806,28 @@ const DOMUtils = (doc: Document, settings: Partial<DOMUtilsSettings> = {}): DOMU
     unbind: unbind as DOMUtils['unbind'],
 
     /**
-     * Fires the specified event name with object on target.
-     *
+     * Fires the specified event on the specified target.
+     * <br>
+     * <em>Deprecated in TinyMCE 6.0 and has been marked for removal in TinyMCE 7.0. Use the <code>dispatch<code> instead.</em>
      * @method fire
-     * @param {Node/Document/Window} target Target element or object to fire event on.
-     * @param {String} name Name of the event to fire.
+     * @param {Object} target Target node/window or custom object.
+     * @param {String} name Event name to fire.
+     * @param {Object} args Optional arguments to send to the observers.
+     * @return {EventUtils} Event utils instance.
+     * @deprecated Use dispatch() instead
+     */
+    fire,
+
+    /**
+     * Dispatches the specified event name with object on target.
```

Comment:
I am not sure on the exact wording but maybe something like this?
```suggestion
     * Dispatches the specified event name and optional object on the specified target.
```